lib/data_requests/local/fetch_cloudwatch_logs.rb

```diff
@@ -80,7 +80,7 @@ def cloudwatch_client
       def query_string
         <<~QUERY
           fields @timestamp, @message
-          | filter properties.user_id = '#{uuid}' and name != 'IRS Attempt API: Event metadata'
```

Comment:
keep this in the query, since the old metadat events have already been logged, and will already muck up the results

Reply:
> keep this in the query, since the old metadat events have already been logged, and will already muck up the results

Yeah I waffled on this too. I didn't look closely enough at how this class was used and sorta assumed it was just for ad-hoc querying, but I get the impression we're also doing some historical querying with this class as well?

So, yeah, to your point, makes sense to keep.

Reply:
Reverted in 6f868564d.

Reply:
This is the query that is used as part of OIG requests, so yes, for historical querying